bindings/wasm/src/account/wasm_account.rs

```diff
@@ -0,0 +1,320 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::credential::WasmCredential;
+use crate::credential::WasmPresentation;
+use crate::crypto::WasmSignatureOptions;
+use crate::did::PromiseResolvedDocument;
+use crate::did::WasmDID;
+use crate::did::WasmDocument;
+use crate::did::WasmResolvedDocument;
+use crate::error::Result;
+use crate::error::WasmResult;
+use identity::account::Account;
+use identity::account::AccountBuilder;
+use identity::account::AccountStorage;
+use identity::account::PublishOptions;
+use identity::account::Storage;
+
+use identity::crypto::SetSignature;
+use identity::crypto::SignatureOptions;
+use identity::did::verifiable::VerifiableProperties;
+use identity::iota::IotaDID;
+use identity::iota::IotaDocument;
+
+use js_sys::Promise;
+
+use crate::account::wasm_auto_save::WasmAutoSave;
+use std::rc::Rc;
+use std::sync::Arc;
+use wasm_bindgen::__rt::Ref;
+use wasm_bindgen::__rt::WasmRefCell;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen::JsCast;
+use wasm_bindgen_futures::future_to_promise;
+
+/// An account manages one identity.
+///
+/// It handles private keys, writing to storage and
+/// publishing to the Tangle.
+#[wasm_bindgen(js_name = Account)]
+pub struct WasmAccount(pub(crate) Rc<WasmRefCell<Account>>);
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  #[wasm_bindgen(js_name = did)]
+  pub fn did(&self) -> WasmDID {
```

Comment:
Nit: missing doc-comment.

```suggestion
  /// Returns the {@link DID} of the managed identity.
  #[wasm_bindgen(js_name = did)]
  pub fn did(&self) -> WasmDID {
```